### ISSUE TITLE
ocide cannot be started from MINGW artifacts

### ISSUE THREAD

**Issue by GitMensch** [LADSoft/OrangeC#268]
I've downloaded occe.zip from https://ci.appveyor.com/project/LADSoft/orangec/builds/20190245/job/ldy9naub674obd8w/artifacts

Also happens with older artifacts, for example https://ci.appveyor.com/project/LADSoft/orangec/builds/19837495/job/cjro5stn9jv6mpiu/artifacts (15 days ago).

The other executables can be started but ocide cannot be started, error 0xc000007b.
Rechecked: Only MS builds work, I'm quite sure the MinGW build did work before, too.

**Comment by LADSoft** [comment]
if you compile with MINGW64 it creates 64 bit executables.    I rigged it so that OCIDE would compile in this environment, but not run because of all the WIN32 dependencies.    So the executable you reference has never run and it is a known issue.    OCIDE should run in all the other build configurations though.

The last time I tried, compiling with MINGW32 did indeed produce a running OCIDE.

There is already an issue LADSoft/OCIDE#21 for dealing with altering OCIDE to be able to run in 64-bit environment.    But I have no idea what it will take to actually do this.

As a side note there is also an issue LADSoft/OrangeC#111, in which we make 64-bit artifacts and install.
